lakegallery/map/static/map/style.css

```diff
@@ -485,7 +486,16 @@ ol > img {
 }
 
 .modal-body > .close {
+	width: 30px;
+	height: 30px;
+	top: 20px;
+	right: 20px;
 	cursor: pointer;
+	position: absolute;
+	background-color: white;
+	border-radius: 15px;
+  color: black;
```

Comment:
align?

Reply:
i saw that. not sure why it shows that way. my style sheet code looks clean.
![image](https://user-images.githubusercontent.com/13525011/64698618-d6fb6e00-d468-11e9-9d1b-d1c13ca5b283.png)

Reply:
oh, that is weird. i approve!